src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -1470,18 +1470,25 @@ private bool IsXamlTypeConverter(INamedTypeSymbol symbol)
 		private string BuildXamlTypeConverterLiteralValue(INamedTypeSymbol symbol, string memberValue, bool includeQuotations)
 		{
 			var attributeData = symbol.FindAttribute(XamlConstants.Types.CreateFromStringAttribute);
-			var returnType = attributeData?.NamedArguments.FirstOrDefault(kvp => kvp.Key == "MethodName").Value.Value;
+			var returnType = attributeData?.NamedArguments.FirstOrDefault(kvp => kvp.Key == "MethodName").Value.Value.ToString();
+
+			// Since the MethodName value can simply be the name of the method
+			// without its full path, example: nameof(MyConversionMethod), we must
+			// make sure to fully qualify the method name with its namespace
+			var fullyQualifiedReturnType = !returnType.Contains('.')
```

Comment:
Should we try to resolve the type instead of checking for the presence of a `.` ?

Reply:
Tried it but realized it can`t be done. The attribute value points to a method and not a type. Instead I`ve renamed variables for better clarity of what`s happening.

Reply:
I agree with Jérôme here...  having a dot is not enough to be considered as fully qualified. Could be a nested class or a relative namespace.

Reply:
@jeromelaban  @carldebilly  Suppose we have 

`[Windows.Foundation.Metadata.CreateFromString(MethodName = "CreateFromStringTest.Location.ConvertToLatLong")]`

or

`[Windows.Foundation.Metadata.CreateFromString(MethodName = nameof(ConvertToLatLong))]`

Using a `FindType` or `GetType` on the `MethodName` parameter won't work, so checking for `.` was used. Is there another way of validating it as a fully qualified value?

Reply:
Could you try to resolve it using `GetType()`?

Reply:
@carldebilly  I did, I used FindType since GetType will throw an exception. FindType always returns null for a valid namespace 😐  But it makes sense since the namespace path contains a function:

ex: `FindType("CreateFromStringTest.Location.ConvertToLatLong")`